### ISSUE TITLE
Processing

### ISSUE THREAD

**Issue by 3nids** [FloEll/QWaterModel#3]
Have you considered making the plugin a processing algorithm ?
This sounds like a very suitable candidate.

**Comment by FloEll** [comment]
no, not yet. But thanks for the suggestion, I'll have a look into it.
